src/components/Autocomplate/Autocomplate.tsx

```diff
@@ -0,0 +1,92 @@
+import classNames from 'classnames';
+import { useMemo, useState, useCallback } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
you should add it
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/87462af8-03c8-4100-82b7-c46e9c039979)

or maybe you forget to add changes in the package.json to git

Reply:
still not fixed
<img width="626" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/70220667/d8dafc5d-8b44-49fe-94c1-1b78ca810ec2">
you need to write `gid add package.json`